edumap/config/initializers/contact_us.rb

```diff
@@ -0,0 +1,34 @@
+# Use this hook to configure contact mailer.
+ContactUs.setup do |config|
+
+  # ==> Mailer Configuration
+
+  # Configure the e-mail address which email notifications should be sent from.  If emails must be sent from a verified email address you may set it here.
+  # Example:
+  # config.mailer_from = "contact@please-change-me.com"
+  config.mailer_from = nil
+
+  # Configure the e-mail address which should receive the contact form email notifications.
+  config.mailer_to = "edumapcs@gmail.com" #change this to contact email
```

Comment:
I think that if we add emailing, it'd be a good idea to background it / run it asynchronously. Otherwise it could block regular usage of the site. Maybe [Suckerpunch](https://github.com/brandonhilkert/sucker_punch) would be useful here, since it doesn't require us to set up any Sidekiq or Redis servers.